west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: d7af6f371034f31b9440b27c694b0be3c87491d3
+      revision: 6a0b1da158a4f8bc5f2ca5058637dce26a38660e
```

Comment:
can you make this a separate patch for bisect-ability.

Reply:
I think that if we separate the west update into a separate commit, then we'll lose the ability to bisect because these changes won't compile without updating Zephyr.
@RanderWang  correct me if I'm wrong.

Reply:
@lgirdwood west update needs to be in same commit, otherwise bisect is broken.